project_pivot/views/project_project.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!--
+    Copyright 2024 Tecnativa - Carolina Fernandez
+    License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+-->
+<odoo>
+    <record id="project.open_view_project_all" model="ir.actions.act_window">
+        <field name="view_mode">kanban,tree,form,pivot</field>
```

Comment:
Sorry, but I don't see the pivot view in this action.
Please check if this module might be incompatible.

https://github.com/OCA/project/blob/16.0/project_timeline/views/project_project_view.xml#L18

![image](https://github.com/user-attachments/assets/02f80ff1-bafd-408d-94fe-fea6b947d01f)

Reply:
Well, there's here an incompatibility that can't be solved. Document it and nothing more can be done for now.